src/v2/cookbook/debugging-in-vscode.md

```diff
@@ -95,7 +95,7 @@ There are other methods of debugging, varying in complexity. The most popular an
 * Subscribe to specific parts of your application’s state for faster debugging
 * Visualize component hierarchy, and an animation allows you to collapse or expand the tree for specific hierarchy views.
 
-![Vuetron Heirarchy](/images/vuetron-heirarchy.gif)
+![Vuetron Hierarchy](/images/vuetron-hierarchy.gif)
```

Comment:
The corresponding image file should be renamed as well.

Reply:
It has been renamed:
https://github.com/vuejs/vuejs.org/blob/218e7138bc247db53a89364986f52076c279692f/src/images/vuetron-hierarchy.gif